ibis/expr/operations/reductions.py

```diff
@@ -210,13 +210,25 @@ class Median(QuantileBase):
     """Compute the median of a column."""
 
 
+@public
+class ApproxMedian(Median):
```

Comment:
Any reason to subclass `Median` here instead of `QuantileBase`?

Reply:
Small refactor to ensure all approximate methods have consistent typing with their non-approximate versions.